### PR TITLE
SOC-1463 | Introduce ember-pop-over component

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/SOC-1463

### CODE REVIEW

File: front/main/app/styles/main/component/_pop-over.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	position: absolute;
+	z-index: $z-9;
+}
+
+.pop-over-compass-invisible {
+	visibility: hidden;
+}
+
+.pop-over-container {
+	background-color: $color-white;
```
Comment (bkoval) [inline front/main/app/styles/main/component/_pop-over.scss]: I believe we should keep color variables in _settings.scss to be able to override them easily (e.g. for dark theme purposes)

File: front/main/app/styles/main/component/_pop-over.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	border-radius: 2px;
+	box-shadow: 0 4px 12px rgba(0, 0, 0, .3);
+	padding: 20px;
+}
+
+.pop-over-pointer {
+	border: $pointer-size / 2 solid transparent;
+	display:  block;
```
Comment (bkoval) [inline front/main/app/styles/main/component/_pop-over.scss]: Two spaces! :o

File: front/main/app/styles/main/component/_pop-over.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+	height: 0;
+	pointer-events: none;
+	position: absolute;
+	width: 0;
+	z-index: 2;
+}
+
+.pop-over-pointer.orient-above {
+	border-color: $color-white transparent transparent transparent;
```
Comment (bkoval) [inline front/main/app/styles/main/component/_pop-over.scss]: https://github.com/Wikia/mercury/pull/1841/files#diff-510bd5d74ce4a3a9a9582629e6b8f7c5R17

File: front/main/app/styles/main/component/_pop-over.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+.pop-over-pointer {
+	border: $pointer-size / 2 solid transparent;
+	display: block;
+	height: 0;
+	pointer-events: none;
+	position: absolute;
+	width: 0;
+	z-index: 2;
```
Comment (bkoval) [inline front/main/app/styles/main/component/_pop-over.scss]: Is there no style-guide value to use here?

**Comment by rafalkalinski** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM, but let's try to make the tests pass :)

**Comment by bkoval** [comment]
LGTM :page_facing_up:

File: front/main/package.json [hunk 5]
```diff
@@ -44,6 +44,7 @@
     "ember-derequire": "0.1.2",
     "ember-disable-proxy-controllers": "1.0.1",
     "ember-export-application-global": "1.0.4",
+    "ember-pop-over": "git+https://github.com/kvas-damian/ember-pop-over#Fix-touch-devices",
```
Comment (rogatty) [inline front/main/package.json]: If used like this, npm will download it every time you use `npm install`.
If https://github.com/tim-evans/ember-pop-over/pull/23 won't be merged before you want to merge this to dev, you should use `kvas-damian/ember-pop-over#Fix-touch-devices`. You probably need to create a release tag, too.

**Pull request closed without merging**